package.json

```diff
@@ -640,11 +636,6 @@
                     "when": "view == azFuncTree && viewItem == deployment/github",
                     "group": "1@3"
                 },
-                {
```

Comment:
Wait, we're getting rid of the icon?  I thought we'd have both the double-click and the button.

Reply:
https://github.com/microsoft/vscode-azurefunctions/issues/1862 is about becoming more consistent with VS Code patterns and I can't find any pattern that uses clicking a leaf node and a button for the same functionality

Reply:
I think people already have a hard enough time finding the deployment logs.  I think we need a way to make it obvious that a node can be double-clicked if we do this.

Reply:
First of all it's a single click, not a double click. Secondly, the argument made in https://github.com/microsoft/vscode-azurefunctions/issues/1793 was that people _did_ expect a single click to do something and it wasn't. That's how it works in the file view, git view, etc.

I agree we probably want to make deployment logs more prominent somehow, but that was not the goal of this change. The goal was to have more consistent behavior for leaf nodes. We can continue to discuss the other piece here: https://github.com/microsoft/vscode-azureappservice/issues/1251